nixos/modules/system/boot/stage-1.nix

```diff
@@ -131,6 +131,7 @@ let
 
       # Copy udev.
       copy_bin_and_libs ${udev}/bin/udevadm
+      cp ${lib.getLib udev.kmod}/lib/libkmod.so* $out/lib
```

Comment:
Fun fact: `CONFIG_MODPROBE_PATH`, when the kernel is built, controls what this path is.

Reply:
That's for the `modprobe` binary.

Reply:
Right you are.